docs/self_hosting/configuration/external_clickhouse.mdx

```diff
@@ -79,9 +81,19 @@ CLICKHOUSE_PORT=8123 # Change to your Clickhouse port if needed
 CLICKHOUSE_TLS=false # Change to true if you are using TLS to connect to Clickhouse. Otherwise, leave it as is
 CLICKHOUSE_PASSWORD=password # Change to your Clickhouse password if needed
 CLICKHOUSE_NATIVE_PORT=9000 # Change to your Clickhouse native port if needed
+CLICKHOUSE_CLUSTER=my_cluster_name # Optional: Set this if using a clustered deployment
 `
     ),
   ]}
 />
 
 Once configured, you should be able to reinstall your LangSmith instance. If everything is configured correctly, your LangSmith instance should now be using your external ClickHouse database.
+
+:::note
+When using a clustered deployment, LangSmith will automatically:
```

Comment:
Should we mention this only works with an "external" clickhouse?  Right now we don't add the keeper configurations, so wouldn't work with our k8s setup (@langchain-infra if any specific comments here).